src/CompanyService.Data.Provider.MsSql.Ef/Migrations/20210601145730_AddDbPositionUser.cs

```diff
@@ -0,0 +1,58 @@
+﻿using LT.DigitalOffice.CompanyService.Models.Db;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+
+namespace LT.DigitalOffice.CompanyService.Data.Provider.MsSql.Ef.Migrations
+{
+    [DbContext(typeof(CompanyServiceDbContext))]
+    [Migration("20210601145730_AddDbPositionUser")]
+    public class AddDbPositionUser : Migration
+    {
+        private void CreatePositionUsersTable(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: DbPositionUser.TableName,
+                columns: table => new
+                {
+                    Id = table.Column<Guid>(nullable: false),
+                    PositionId = table.Column<Guid>(nullable: false),
+                    UserId = table.Column<Guid>(nullable: false),
+                    IsActive = table.Column<bool>(nullable: false),
+                    StartTime = table.Column<DateTime>(nullable: false),
+                    EndTime = table.Column<DateTime?>(nullable: true)
```

Comment:
Why this need?

Reply:
What? endtime? it was in models of department user + i don't think that it is unnecessary info